hs_collection_resource/urls.py

```diff
@@ -6,4 +6,8 @@
     url(r'^_internal/(?P<shortkey>[A-z0-9]+)/update-collection/$', views.update_collection),
     url(r'^_internal/(?P<shortkey>[A-z0-9]+)/update-collection-for-deleted-resources/$',
         views.update_collection_for_deleted_resources),
+    url(r'^_internal/(?P<shortkey>[A-z0-9]+)/update-collection-coverages/$',
```

Comment:
@pkdash 
The current url pattern of "update-collection-coverages"  and "calculate-collection-coverages" conforms to the pattern of "update-collection" and "update-collection-for-deleted-resources".